custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -190,7 +190,9 @@ def updates(self):
            cases_person_adolescent_girls_11_14_all = ut.cases_person_adolescent_girls_11_14_all,
            cases_person_adolescent_girls_15_18 = ut.cases_person_adolescent_girls_15_18,
            cases_person_adolescent_girls_15_18_all = ut.cases_person_adolescent_girls_15_18_all,
-           cases_person_referred = ut.cases_person_referred
+           cases_person_referred = ut.cases_person_referred,
+           cases_person_adolescent_girls_11_14_out_of_school = ut.girls_out_of_schoool,
+           cases_person_adolescent_girls_11_14_all_new = ut.cases_person_adolescent_girls_11_14_all_new
```

Comment:
Actually change the columns in agg_Awc

Reply:
`*_new` is generally a bad name because eventually this column will be old. something like `*_v2` is a bit more future proof.

Is this column expected to completely take over for the other column with the same name or is there a more descriptive name that could be?

Reply:
Yeah, So these new column will going to take over the others with similar name in future(probably right after QA is finished).
Sure. `_v2` makes mroe sense. Doing this.